nexus_client_sdk/nexus/core/serializers.py

```diff
@@ -0,0 +1,70 @@
+"""Serialization format module."""
+from typing import final, Any, TypeVar, Type
+
+import pandas
+from adapta.storage.models.format import SerializationFormat
+from adapta.storage.models.formatters import (
+    PandasDataFrameParquetSerializationFormat,
+    DictJsonSerializationFormat,
+)
+
+T = TypeVar("T")  # pylint: disable=C0103
```

Comment:
This name should not be invalid according to docs: https://pylint.readthedocs.io/en/stable/user_guide/messages/convention/invalid-name.html#predefined-naming-patterns